pkg/volume/volume.go

```diff
@@ -181,6 +181,9 @@ type BlockVolumeUnmapper interface {
 	// If the plugin is non-attachable, this method detaches the volume
 	// from a node.
 	TearDownDevice(mapPath string, devicePath string) error
+
+	// UnmapDevice unmaps the block device path
+	UnmapDevice(globalUnmapPath, podDeviceUnmapPath, volName string, podUID types.UID) error
```

Comment:
I am thinking that we should call this function `UnmapPodDevice` or something like that - so as it is evident that this function is actually going to remove device/volume for the pod and not from global mount path.

Reply:
Then, do we need to also change `MapDevice` to `MapPodDevice`?

If everyone agrees, I will go ahead and change them at once.

Reply:
Agree to rename. It's quite confusing because there is the UnmapDevice operation, UnmapDevice util function, and here the UnmapDevice method for BlockVolume plugins.

Reply:
Renamed. Thank you for your explanation on the reason for rename.